src/repositories/issue.repository.ts

```diff
@@ -0,0 +1,45 @@
+import { connect } from '../config/db.config';
+import { Issue } from '../models/issue.model';
+
+export class IssueRepository {
+  private db: any = {};
+  private issueRepository: any;
+
+  constructor() {
+    this.db = connect();
+
+    this.db.sequelize.sync({ force: true });
+    this.issueRepository = this.db.sequelize.getRepository(Issue);
```

Comment:
this is not needed, you can just call the model `Issue`. For example, `Issue.findByPk(1)`